content/syllabuses/tilde-intro-tech-student.md

```diff
@@ -9,6 +9,6 @@ The Tilde learning platform is not 100% intuitive to everyone from the get go. T
 - {{% contentlink path="topics/bootcamp/success-in-bootcamp" %}}
 - {{% contentlink path="topics/bootcamp/failure-at-bootcamp" %}}
 - {{% contentlink path="projects/tilde/link-card-tutorial" flavour=""%}}
-- {{% contentlink path="projects/tilde/repo-card-tutorial/part-2" flavour=""%}}
-- {{% contentlink path="topics/tilde/understanding-the-review-column" %}}
```

Comment:
Normal repo cards (and the whole PR review workflow) are only needed after the main BootCamp content so I took it out of the intro.